kola/tests/docker/docker.go

```diff
@@ -206,6 +212,12 @@ systemd:
   units:
    - name: docker.service
      enable: true`),
+
+		// <link to mantle bug>
```

Comment:
Missing link here?

Reply:
I was originally intending to post up the issue and force-push an amended commit that had the correct links but I'll post it later and update.